dev-docs/show-outstream-video-ads.md

```diff
@@ -178,6 +183,48 @@ For more information, see the API documentation for:
 + [getAdserverTargetingForAdUnitCode]()
 + [renderAd]()
 
+## Option 3. With a Publisher-defined Renderer
+
+By default, an outstream bid response is accompanied by a video renderer in the form of a URL that points to a script containing the renderer software.  This software is used by Prebid.js to load and play the winning outstream bid.
```

Comment:
If the adunit has a publisher-defined renderer associated, the demand partner bid response should not need to include any renderer.